ParcelDistributionCenter/ParcelDistributionCenter.Web/Controllers/PackagesController.cs

```diff
@@ -61,6 +62,34 @@ public ActionResult DeletePackage(int packageNumber)
             TempData["MessageClass"] = "alert-danger";
             return RedirectToAction(nameof(DisplayPackages));
         }
+      
+        // GET: ProductsController/Delete/5
+        public ActionResult Delete(int id)
+        {
+            var model = _packageHandler.FindPackageByNumber(id);
+            return View(model);
+        }
+
+        // POST: ProductsController/Delete/5
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Delete(int id, Package model)
```

Comment:
Tutaj podobnie co wyżej.

Reply:
Nie wiem też, czy dobrym jest to, że jest to w try catchu. Chyba się mieliśmy o to dopytać, czy się tak robi. Wydaje mi się, że to może zużywać trochę pamięci niepotrzebnie i lepiej zrobić jakąś walidację jak w innych metodach

Reply:
Sprawdzę tego try catcha czy jest potrzebny. Korzystałem z wygenerowanych wzorców i tym się zasugerowałem.